src/main/java/io/mosip/mimoto/service/impl/CredentialServiceImpl.java

```diff
@@ -126,7 +130,7 @@ public ByteArrayInputStream downloadCredentialAsPDF(String issuerId, String cred
         boolean verificationStatus = issuerId.toLowerCase().contains("mock") || verifyCredential(vcCredentialResponse);
         if(verificationStatus) {
             String dataShareUrl =  QRCodeType.OnlineSharing.equals(issuerConfig.getQr_code_type()) ? dataShareService.storeDataInDataShare(objectMapper.writeValueAsString(vcCredentialResponse), credentialValidity) : "";
-            return generatePdfForVerifiableCredentials(vcCredentialResponse, issuerConfig, credentialsSupportedResponse, dataShareUrl, credentialValidity);
+            return generatePdfForVerifiableCredentials(issuerId, credentialType,vcCredentialResponse, issuerConfig, credentialsSupportedResponse, dataShareUrl, credentialValidity, locale);
```

Comment:
For locale story changes we don't need issuerId and credential type to get added in this method. So remove them if not needed.

Reply:
This issuerId & crednetialType to support multiple templates in future, where name of template would include issuerId & credentialType.
Should It be kept or removed?